tests/conftest.py

```diff
@@ -210,8 +219,17 @@ def pytest_sessionfinish(
     if get_xdist_worker_id(session) != "master":
         return
 
-    lock_file = session.stash[fixture_lock]
-    session.stash[fixture_lock] = None
```

Comment:
Do we want to avoid locking when running under mutmut?

Reply:
I suppose it depends how mutmut runs in parallel. If it's entirely separate processes, without this lock, each one would blow away the fixtures directory when it starts.